Part2_LinuxKernelBootOptions.html

```diff
@@ -283,11 +283,11 @@ <h3>pci=nomsi</h3>
 igb 0000:05:00.1: eth1: PBA No: 106300-000
 igb 0000:05:00.1: Using legacy interrupts. 1 rx queue(s), 1 tx queue(s)</pre></source>
 	
-	<p>Becuase of the decreased number of interrupts per device, enabling such option can lead to the significant performance limitation of the device driver (and it is not even counting that according to the Intel research <a href="https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/msg-signaled-interrupts-paper.pdf">'Reducing Interrupt Latency Through the Use of Message Signaled Interrupts'</a> MSI interrupts 3 times faster than the IO-APIC interrupts and 5 times faster than the PIC interrupts.<p/>
+	<p>Because of the decreased number of interrupts per device, enabling this option can lead to a significant performance limitation of the device driver, and that is not even counting that according to the Intel research <a href="https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/msg-signaled-interrupts-paper.pdf">'Reducing Interrupt Latency Through the Use of Message Signaled Interrupts'</a>, MSI interrupts three times faster than the IO-APIC interrupts and five times faster than the PIC interrupts.<p/>
```

Comment:
Can we stay with numbers?
"MSI interrupts 3 times faster than the IO-APIC interrupts and 5 times faster than the PIC interrupts"

I think it is harder to get this important quantity information when it is just a big sentence of text. But the numbers draw the attention and make it easier for the reader to process it.

Reply:
Yup, writing the numbers out is good too.